.github/workflows/strain_dispatcher.yml

```diff
@@ -34,7 +34,7 @@ jobs:
               "base_ref": "${{ github.base_ref }}",
               "strain_property": "${{ env.STRAIN_PROPERTY }}",
               "strain_property_value": "${{ env.PROPERTY_VALUE }}",
-              "strain_path": "palm/nelp/strain.yml",
+              "strain_path": "${{ env.STRAIN_PATH }}",
```

Comment:
I think here you have 2 options:

- Use the repo vars directly ${{ vars.STRAIN_PATH }}

- Add the repo vars to env vars in this line:
https://github.com/eduNEXT/eox-nelp/pull/148/commits/54a8f57444fe8fc6e7bff041734199fae3c2ccaf#diff-d8437808a5e13e732734dc93812f518c2a153b9acc3fd92498dcfe2c217a80b3R15-R17